client/app/pages/queries-list/QueriesListEmptyState.jsx

```diff
@@ -19,11 +20,11 @@ export default function QueriesListEmptyState({ page, searchTerm, selectedTags }
     case "my":
       return (
         <div className="tiled bg-white p-15">
-          <a href="queries/new" className="btn btn-primary btn-sm">
+          <Link.Button href="queries/new" className="btn btn-primary btn-sm">
```

Comment:
Should we remove these classNames and use Antd styling?

Reply:
:man_facepalming: yes, sure, thanks!

Reply:
![image](https://user-images.githubusercontent.com/12139186/91281751-24174380-e791-11ea-8e81-7f00c321b15f.png)

Reply:
https://github.com/getredash/redash/pull/5122/commits/072b2be2c9dc5f7db56594fd1616bffc84a86576

Reply:
The sizing was also small, do you think it looks better with normal?

Reply:
To be honest - I'm not sure which is better:

![image](https://user-images.githubusercontent.com/12139186/91303304-7caa0900-e7b0-11ea-838f-d76f05fdc3f5.png)

WDYT - should it be small or default size? I really cant chose :slightly_smiling_face:

Reply:
Let's leave it small as it used to be like that. That page probably could use a different empty state, but that's out of scope here :)

Reply:
https://github.com/getredash/redash/pull/5122/commits/0d13c530a6d64768b283005437e776122a71e23d

Also I renamed `value-format.js` to `.jsx` as now it contains some JSX fragments